docs/PRODUCTION.md

```
@@ -0,0 +1,425 @@
+\`\`\`
+
+### Database maintenance
+
+\`\`\`bash
+sqlite3 /var/lib/litelog/litelog.db "VACUUM;"
+\`\`\`
+
+### Backup script
+
+\`\`\`bash
+#!/bin/bash
```

markdownlint-cli2 (0.21.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)